### ISSUE TITLE
[Bug Report] Rendering issue in LunarLander-v2

### ISSUE THREAD

**Issue by zhaoyi11** [openai/gym#3052]
**Describe the bug**
When can render() in the LunaerLander_v2 environment (gym version 0.25.2), I got this "pygame.transform.scale(self.surf, (SCALE, SCALE)). TypeError: integer argument expected, got float.". The error corresponds to this line https://github.com/openai/gym/blob/e89b00c363cf26cfe3f5db7b4320cc30eb2ca7b9/gym/envs/box2d/lunar_lander.py#L590 

**Code example**
```python
import gym
env = gym.make("LunarLannder-v2", render_mode="rgb_array")
env.reset()
```

**System Info**
- I install Gym (v0.25.2) by pip.
- I use Linux.
- Python version 3.8.12

**Additional context**
I fix this issue by change this line https://github.com/openai/gym/blob/e89b00c363cf26cfe3f5db7b4320cc30eb2ca7b9/gym/envs/box2d/lunar_lander.py#L590 to
```python
pygame.transform.scale(self.surf, (int(SCALE), int(SCALE)))
```
bipedal_walker should have the similar issue 
https://github.com/openai/gym/blob/ab3e02db83a020cc8ad9b972767ba75274f29188/gym/envs/box2d/bipedal_walker.py#L629

### Checklist

- [x ] I have checked that there is no similar [issue](https://github.com/openai/gym/issues) in the repo (**required**)

**Comment by RedTachyon** [comment]
I can't seem to replicate this. What pygame version are you using? I'm guessing that might be the issue.

**Comment by zhaoyi11** [comment]
Thanks for your reply. Yes, it's because of the pygame version. I met the above problem with pygame v2.0.1. And the problem is fixed by updating it to v2.1.2. I will close the issue.
